src/styles/card.scss

```diff
@@ -0,0 +1,129 @@
+@import 'variables';
+
+.container {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: 1fr;
+  gap: 46px 48px;
+  padding: 50px 40px;
+
+  @media (max-width: 487px) {
+    grid-template-columns: 1fr;
+  }
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  margin: 0;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  background-color: $background-color;
+  border-radius: $border-radius;
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The hover effect on the `.card` class uses `transform: scale(1.2)`, which can cause layout shifts and affect the user experience. Consider using a more subtle transformation or adding `will-change: transform;` to optimize performance and prevent layout shifts.